react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -1303,9 +1307,13 @@ exports[`Menu should render examples: Menu 3`] = `
 
 exports[`MuiCozyTheme/Buttons should render examples: MuiCozyTheme/Buttons 1`] = `"<div></div>"`;
 
-exports[`MuiCozyTheme/ExpansionPanel should render examples: MuiCozyTheme/ExpansionPanel 1`] = `"<div></div>"`;
+exports[`MuiCozyTheme/List should render examples: MuiCozyTheme/List 1`] = `"<div></div>"`;
+
+exports[`MuiCozyTheme/List should render examples: MuiCozyTheme/List 2`] = `"<div></div>"`;
+
+exports[`MuiCozyTheme/List should render examples: MuiCozyTheme/List 3`] = `"<div></div>"`;
 
-exports[`MuiCozyTheme/ExpansionPanel should render examples: MuiCozyTheme/ExpansionPanel 2`] = `"<div></div>"`;
+exports[`MuiCozyTheme/List should render examples: MuiCozyTheme/List 4`] = `"<div></div>"`;
```

Comment:
Our snapshots doesn't seem to be very useful :thinking: (not specifically related to this PR, but I am just paying attention to this).